cloudify_rest_client/blueprints.py

```diff
@@ -225,6 +262,48 @@ def upload(self,
         finally:
             shutil.rmtree(tempdir)
 
+    def validate(self,
+                 path,
+                 entity_id,
+                 visibility=VisibilityState.TENANT,
+                 progress_callback=None,
+                 skip_size_limit=True):
+        """
+        Validates a blueprint with Cloudify's manager.
+
+        :param path: Main blueprint yaml file path.
+        :param entity_id: Id of the uploaded blueprint.
+        :param visibility: The visibility of the blueprint, can be 'private',
+                           'tenant' or 'global'.
+        :param progress_callback: Progress bar callback method
+        :param skip_size_limit: Indicator whether to check size limit on
+                           blueprint folder
+        :return: Created response.
+
+        Blueprint path should point to the main yaml file of the response
+        to be uploaded. Its containing folder will be packed to an archive
+        and get uploaded to the manager.
+        Validation is basically an upload without storage part being done.
```

Comment:
s/without storage/without **the** storage/

Reply:
ok